### ISSUE TITLE
Official train args for yolo26 model?

### ISSUE THREAD

**Issue by wufan-tb** [ultralytics/ultralytics#24654]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I trained a yolo26l model with default args, (only changed the device=0,1,2,...8, and batchsize=512), and the best weights got mAP(50-95) = 0.494, significantly lower than official metrics (mAP(50-95) = 0.55) (on coco val set). So,  with which args can I get a same model with official one.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @wufan-tb, thank you for your interest in Ultralytics 🚀! This is an automated response to help get things moving, and an Ultralytics engineer will also assist you soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and where many of the most common questions may already be answered.

Since this looks like a training/results ❓ question, please provide as much information as possible, including your dataset setup, exact training command or script, image size, epochs, optimizer/settings, batch size, device configuration, evaluation command, and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips) 🧪

If this turns out to be a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
@wufan-tb https://platform.ultralytics.com/ultralytics/yolo26/yolo26n for training args. You can copy all args as a JSON from the tables:

<img width="1534" height="1214" alt="Image" src="https://github.com/user-attachments/assets/d3bea9a0-f727-4cb6-98b2-ed485511c35b" />

**Comment by wufan-tb** [comment]
Thanks, I found the pretrained model is `yolo26l-objv1-150.pt`, which dataset was this model pretrained on, and where can I find the download link?

or can I get a train-from-scratch best-hyp args?

**Comment by lmycross** [comment]
It's trained on [objects365](https://docs.ultralytics.com/datasets/detect/objects365#key-features) dataset, the pre-trained weights can be found at: https://github.com/ultralytics/assets/releases/tag/v8.4.0

**Comment by wufan-tb** [comment]
> [@wufan-tb](https://github.com/wufan-tb) https://platform.ultralytics.com/ultralytics/yolo26/yolo26n for training args. You can copy all args as a JSON from the tables:
> 
> <img alt="Image" width="1534" height="1214" src="https://private-user-images.githubusercontent.com/26833433/600734679-d3bea9a0-f727-4cb6-98b2-ed485511c35b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODA0NTE2ODUsIm5iZiI6MTc4MDQ1MTM4NSwicGF0aCI6Ii8yNjgzMzQzMy82MDA3MzQ2NzktZDNiZWE5YTAtZjcyNy00Y2I2LTk4YjItZWQ0ODU1MTFjMzViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA2MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNjAzVDAxNDk0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3ZjAwY2ZmNGEzMmMwZGE3M2UzNWNlNjVkMWVhOTg5MDcxNTdiYjJkNmVlMjRiMDcwNDZkZTdiN2FmZTc3MDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.Z_OO6sKLS7UvwZTnJUWKdWr9VKGeDp-4qwyPcEo5vU8">

Hi, I copied this args, and changed my ultralytics version to 8.3.222, yet some error appeared during training, it seems some args keys are wrong.  (besides, I tried the latest version, 8.4.60, got this error too.)

<img width="1772" height="209" alt="Image" src="https://github.com/user-attachments/assets/315c048c-7669-43f1-9fcb-afa1d3e374f6" />

<img width="1894" height="319" alt="Image" src="https://github.com/user-attachments/assets/07f4f39d-c585-448c-8f45-1c78c3501f37" />

**Comment by lmycross** [comment]
These unrecognized parameters are only used for experimentation and have already been dropped, you can remove them from the args.
